setup.py

```diff
@@ -28,7 +31,12 @@
     keywords="networking automation vendor-agnostic",
     package_dir={'': libdir},
     packages=packages,
-    install_requires=install_reqs,
+    extras_require={
```

Comment:
@jeremyschulman I altered this to include the requirements directly for each extras module, instead of hunting for the requirements.txt files in different directories. That was causing failures in travis.